meilisearch-http/src/helpers/meilisearch.rs

```diff
@@ -482,12 +491,7 @@ fn calculate_highlights(
         }
     }
 
-    highlight_result
-}
-
-#[cfg(test)]
-mod tests {
-    use super::*;
+    highlight_result } #[cfg(test)] mod tests { use super::*;
```

Comment:
Whooops

Reply:
Ik ik 😁 it's fixed in local, with other things I'm trying to improve before starting review